test/e2e/failover_test.go

```diff
@@ -69,6 +70,17 @@ func failoverDeploymentTest(t *testing.T, f *framework.Framework, ctx *framework
 			MessagingSystem: siddhiv1alpha2.MessagingSystem{
 				Type: "nats",
 			},
+			PVC: corev1.PersistentVolumeClaimSpec{
+				AccessModes: []corev1.PersistentVolumeAccessMode{
+					corev1.ReadWriteOnce,
+				},
+				Resources: corev1.ResourceRequirements{
+					Requests: corev1.ResourceList{
+						corev1.ResourceStorage: *resource.NewQuantity(1*1024*1024*1024, resource.BinarySI),
```

Comment:
Can't we have like a const to make `1*1024*1024*1024` more meaningful (i.e Gibibyte)?

Reply:
Fixed in here https://github.com/siddhi-io/siddhi-operator/pull/125/commits/a335b8d95001bf018bfb86b4c852a459ed5b70a9